File: recovered/src/com/lumiyaviewer/lumiya/voice/common/messages/VoiceAcceptCall.java
package com.lumiyaviewer.lumiya.voice.common.messages;

import android.os.Bundle;
import com.lumiyaviewer.lumiya.voice.common.VoicePluginMessage;
import com.lumiyaviewer.lumiya.voice.common.model.VoiceChannelInfo;
import javax.annotation.Nullable;

/* loaded from: classes.dex */
public class VoiceAcceptCall implements VoicePluginMessage {

    @Nullable
    public final String sessionHandle;
    public final VoiceChannelInfo voiceChannelInfo;

    public VoiceAcceptCall(Bundle bundle) {
        this.sessionHandle = bundle.getString("sessionHandle");
        this.voiceChannelInfo = new VoiceChannelInfo(bundle.getBundle("voiceChannelInfo"));
    }

    public VoiceAcceptCall(@Nullable String str, VoiceChannelInfo voiceChannelInfo) {
        this.sessionHandle = str;
        this.voiceChannelInfo = voiceChannelInfo;
    }

    @Override // com.lumiyaviewer.lumiya.voice.common.VoicePluginMessage
    public Bundle toBundle() {
        Bundle bundle = new Bundle();
        bundle.putString("sessionHandle", this.sessionHandle);
        bundle.putBundle("voiceChannelInfo", this.voiceChannelInfo.toBundle());
        return bundle;
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/voice/common/messages/VoiceAcceptCall.kt
package com.lumiyaviewer.lumiya.voice.common.messages

import android.os.Bundle
import com.lumiyaviewer.lumiya.voice.common.VoicePluginMessage
import com.lumiyaviewer.lumiya.voice.common.model.VoiceChannelInfo

class VoiceAcceptCall : VoicePluginMessage {
    @JvmField
    val sessionHandle: String?
    @JvmField
    val voiceChannelInfo: VoiceChannelInfo

    constructor(bundle: Bundle) {
        this.sessionHandle = bundle.getString("sessionHandle")
        this.voiceChannelInfo = VoiceChannelInfo(bundle.getBundle("voiceChannelInfo"))
    }

    constructor(sessionHandle: String?, voiceChannelInfo: VoiceChannelInfo) {
        this.sessionHandle = sessionHandle
        this.voiceChannelInfo = voiceChannelInfo
    }

    override fun toBundle(): Bundle = Bundle().apply {
        putString("sessionHandle", sessionHandle)
        putBundle("voiceChannelInfo", voiceChannelInfo.toBundle())
    }
}
